CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to Contribute
+
+Contributing to this project entails submitting pull requests (PRs) to this repository. PRs need to be appropriately assigned and labelled.
+
+1. Fork this repository.
+2. Clone your fork of the repository.
+3. When developing, develop and commit in a new, local branch. Leave the protected branches (e.g. develop) alone. They are needed to fetch from upstream.
```

Comment:
It doesn't matter on which branch you develop, because you're working on the fork. Fork doesn't have protected branches as the main repository.

Reply:
Removed intruction.